src/AccordionItem/accordion-item.spec.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import AccordionItemTitle from '../AccordionItemTitle/accordion-item-title';
+import AccordionItemBody from '../AccordionItemBody/accordion-item-body';
+import AccordionItem from './accordion-item';
+
+jest.mock('uuid', () => {
+    return {
+        v4: () => 'mock-uuid-hash',
+    };
+});
+
+jest.mock('../AccordionItemTitle/accordion-item-title', () => {
+    const RealReact = require.requireActual('react');
+    class MockElement extends RealReact.Component {
+        render() {
+            return RealReact.createElement('div', this.props, this.props.children);
+        }
+    }
+    MockElement.accordionElementName = 'AccordionItemTitle';
+
+    return MockElement;
+});
+
+jest.mock('../AccordionItemBody/accordion-item-body', () => {
+    const RealReact = require.requireActual('react');
+    class MockElement extends RealReact.Component {
+        render() {
+            return RealReact.createElement('div', this.props, this.props.children);
+        }
+    }
+    MockElement.accordionElementName = 'AccordionItemBody';
+
+    return MockElement;
+});
```

Comment:
All of this should be replaced by shallow rendering, where there is no need to mock individual components.